[RM Ch07] Restructuring 7.4.1.2 and 7.4.2 (#1983)
cntt-n/CNTT: doc/ref_model/chapters/chapter07.md

<a name="7.4.1.2"></a>
#### 7.4.1.2 The following general security requirements apply to the Cloud Infrastructure

**System Hardening**

> * No login to root on any platform systems (platform systems are those that are associated with the platform and include systems that directly or indirectly affect the viability of the platform) when root privileges are not required.

Reviewer: Suggest strengthening the language
```suggestion
* Adhering to the principle of least privilege, no login to root on any platform systems (platform systems are those that are associated with the platform and include systems that directly or indirectly affect the viability of the platform) when root privileges are not required.
```

**Platform access**
* Restrict traffic to only traffic that is necessary, and deny all other traffic, including traffic from and to 'Back-end'.
* Provide protections between the Internet and any workloads including web and volumetrics attack preventions.
* All host to host communications within the cloud provider network are to be cryptographically protected in transit.
* Use cryptographically-protected protocols for administrative access to the platform.
* Data Centre Operations staff and systems must use management protocols that limit security risk such as SNMPv3, SSH v2, ICMP, NTP, syslog, and TLS.
* Processes for managing platform access control filters must be documented, followed, and monitored.
* Role-Based Access Control (RBAC) must apply for all platform systems access.
* All APIs access must use TLS protocol, including back-end APIs.

**Workload security**
* Restrict traffic to (and from) the workload to only traffic that is necessary, and deny all other traffic.
* Support zoning within a tenant workload - using application-level filtering.

> * Not expose tenant IP address details to another tenant.

Reviewer: ```suggestion
* Not expose tenant internal IP address details to another tenant.
```

* All production workloads must be separated from all non-production workloads including separation between non-hosted non-production external networks.

**Confidentiality and Integrity**
* All data persisted to primary, replica, or backup storage is to be encrypted.

**Monitoring and security audit**
* All platform security logs are to be time synchronised.
* Logs are to be regularly scanned for events of interest.
* The cloud services must be regularly vulnerability and penetration tested.

**Platform provisioning and LCM**
* A platform change management process that is documented, well communicated to staff and tenants, and rigourously followed.
* A process to check change management adherence that is implemented, and rigourously followed.
* An approved system or process for last resort access must exist for the platform.

> * Where there are multiple hosting facilities used in provision of the service, network communications between facilities for the purpose of backup, management, and application communication are cryptographically protected in transit between data centre facilities.

Reviewer: ```suggestion
* Where there are multiple hosting facilities used in the provisioning of a service, network communications between the facilities for the purpose of backup, management, and workload communications are cryptographically protected in transit between data centre facilities.
```

Author: @pgoyal01 Thanks, modifications committed!
